composer/utils/object_store/mlflow_object_store.py

```diff
@@ -0,0 +1,392 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""MLFlow Artifacts object store."""
+
+from __future__ import annotations
+
+import logging
+import os
+import pathlib
+import uuid
+from typing import Callable, List, Optional, Tuple, Union
+
+from composer.utils import dist
+from composer.utils.import_helpers import MissingConditionalImportError
+from composer.utils.object_store.object_store import ObjectStore, ObjectStoreTransientError
+
+__all__ = ['MLFlowObjectStore']
+
+MLFLOW_DATABRICKS_TRACKING_URI = 'databricks'
+MLFLOW_DBFS_PATH_PREFIX = 'databricks/mlflow-tracking/'
+
+DEFAULT_MLFLOW_EXPERIMENT_NAME = 'mlflow-object-store'
```

Comment:
I was thinking that if the object store is being used in isolation, it might be useful to distinguish it under a default experiment with a name dedicated to just being an object store, but that was on a bit of my own whim. More than happy to keep it consistent with the logger

Reply:
i don't have a strong opinion...happy to go with whatever you think is best. most people will not be using the default experiment name anyway.